mediaplatform/models.py

```diff
@@ -372,15 +372,16 @@ def fetched_size(self):
             ).get('size', '0')
         )
 
-    def get_embed_url(self):
+    def get_jwp_embed_url(self):
```

Comment:
I don't think this method is actually used with the changes in api/views.py so it can probably just be removed.